kinto/core/views/version.py

```diff
@@ -46,4 +46,4 @@ def version_view(request):
                 version_view.__json__ = json.load(f)
                 return version_view.__json__  # First one wins.
 
-    raise httpexceptions.HTTPNotFound()
+    raise FileNotFoundError("Version file missing")
```

Comment:
Thank you for your Pull-Request regarding this.
Travis-CI ran your tests here: https://travis-ci.org/Kinto/kinto/builds/450102518

As you can see in the logs here: https://travis-ci.org/Kinto/kinto/jobs/450102519#L581
The version tests case are now failing because we changed the behaviour.

Can you update the test `Kinto/kinto/tests/core/test_views_version.py:29` to reflect your changes?

Reply:
Sorry, I seem to have created a new pull request instead of updating this one. I'm not entirely sure how to do that. But I've changed `Kinto/kinto/tests/core/test_views_version.py:29` to match our new expected behavior.  https://github.com/Kinto/kinto/pull/1862

Again, sorry for the clutter. Is there a way to simply update this pull request? Cheers.

Reply:
Yes just keep add commit to your branch.

Reply:
Thank you. Again, sorry for the clutter. I'll update the new pull request from now on